lib/docusign_rest/client.rb

```diff
@@ -322,19 +322,28 @@ def get_signers(signers, options={})
           accessCode:                            '',
           addAccessCodeToEmail:                  false,
           customFields:                          nil,
-          iDCheckConfigurationName:              nil,
-          iDCheckInformationInput:               nil,
+          idCheckConfigurationName:              signer[:id_check_configuration_name],
+          idCheckInformationInput:               nil,
           inheritEmailNotificationConfiguration: false,
           note:                                  '',
           phoneAuthentication:                   nil,
+          smsAuthentication:                     nil,
           recipientAttachment:                   nil,
           recipientId:                           signer[:recipient_id] || "#{index + 1}",
-          requireIdLookup:                       false,
+          requireIdLookup:                       signer[:require_id_lookup],
```

Comment:
`signer[:require_id_loopup] || false`? to make sure the original code doesn't break anything